Frontend/src/api/RecentPapers.jsx

```
@@ -0,0 +1,103 @@
+    <li className="bg-white shadow-md rounded-lg p-6 flex justify-between items-center border border-gray-200">
+      <div>
+        <h3 className="text-lg font-semibold text-gray-900">{paper.title}</h3>
+        <p className="text-gray-700"><strong>Field:</strong> {paper.research_field}</p>
+        <p className="text-gray-600">{paper.details}</p>
+      </div>
+      {!collabRequested ? (
+        <button 
+          onClick={() => requestCollaboration(Number(paper.id))}
+          className="bg-blue-500 hover:bg-blue-600 text-white font-semibold py-2 px-4 rounded-lg transition duration-300"
+        >
+          Request Collaboration
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)